pyproject.toml

```diff
@@ -41,8 +41,8 @@ dependencies = [
 # These optional dependencies are for building the
 # application as a part of a Github Action.
 ci = [
-    'pyinstaller==6.3.*',
-    'scipy==1.11.*',
+    'pyinstaller==6.4.*',
+    'scipy==1.12.*',
```

Comment:
these, especially `pyinstaller`, will have to be well tested - we had some problems upgrading pyinstaller in the past.

Reply:
I have added a wiki page with the tests:
https://github.com/easyScience/EasyReflectometryApp/wiki/Testing-in-Hand